src/routes/clubs/[clubId]/challenge/[challengeId]/+layout.svelte

```
@@ -1,68 +1,75 @@
+		getHomeActiveChallengesData,
+		getHomeOpenEntriesData,
+		joinChallenge,
+		leaveChallenge
+	} from '$lib/remote/challenge.remote';
+
+	let { children } = $props();
 
-	let { data, children } = $props();
+	const clubId = page.params.clubId ?? '';
+	const challengeId = page.params.challengeId ?? '';
+	const data = await getChallengeLayoutData({ clubId, challengeId });
```

ESLint

[error] Cannot use `await` in deriveds and template expressions, or at the top level of a component, unless the `experimental.async` compiler option is `true`
https://svelte.dev/e/experimental_async(experimental_async)

(svelte/valid-compile)